airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1362 @@
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: list[google.cloud.dlp_v2.types.JobTrigger]
+        """
+
+        client = self.get_conn()
+
+        if not parent:
+            raise AirflowException("Please provide the parent resource name.")
+
+        results = client.list_job_triggers(
+            parent=parent,
+            page_size=page_size,
+            order_by=order_by,
+            filter_=results_filter,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+        return list(results)
+
+    def list_stored_info_types(
+        self,
+        parent,
+        page_size=None,
+        order_by=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        Lists stored infoTypes.
+
+        :param parent: The parent resource name.
+        :type parent: str
+        :param page_size: (Optional) The maximum number of resources contained in the
+            underlying API response.
+        :type page_size: int
+        :param order_by: (Optional) Optional comma separated list of fields to order by,
+            followed by asc or desc postfix.
+        :type order_by: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: list[google.cloud.dlp_v2.types.StoredInfoType]
+        """
+
+        client = self.get_conn()
+
+        if not parent:
+            raise AirflowException("Please provide the parent resource name.")
+
+        results = client.list_stored_info_types(
+            parent=parent,
+            page_size=page_size,
+            order_by=order_by,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+        return list(results)
+
+    @GoogleCloudBaseHook.catch_http_exception
+    def redact_image(
+        self,
+        parent,
+        inspect_config=None,
+        image_redaction_configs=None,
+        include_findings=None,
+        byte_item=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        Redacts potentially sensitive info from an image. This method has limits on
+        input size, processing time, and output size.
+
+        :param parent: The parent resource name.
+        :type parent: str
```

Comment:
It is worth adding the possibility of defining the project ID using the connection configuration.